### PR TITLE
EA: Remove POST info from Service Facebook Token

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#66664](https://central.tri.be/issues/66664)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [New.php:132](https://github.com/moderntribe/the-events-calendar/blob/a1724dd0a94f50ae2f3920bafd76541f7b4f1650/src/Tribe/Aggregator/Tabs/New.php#L132)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Tabs/New.php [hunk 1]
```diff
@@ -81,7 +81,10 @@ public function get_label() {
 	}
 
 	public function handle_submit() {
-		if ( ! empty( $_POST['ea-facebook-credentials'] ) ) {
+		/**
+		 *
```
Comment (borkweb) [inline src/Tribe/Aggregator/Tabs/New.php]: Add comment text here or remove this comment block

Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/New.php]: Ready for another look

**Comment by borkweb** [comment]
:+1:

**Pull request merged**